Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloud-hypervisor"
-version = "0.14.0"
+version = "15.0.0"
```

Comment:
Question: do we intend to use version number as `15.0.0` or `15.0`? The rest of the article is referring to the latter one.

Reply:
Sadly Rust requires it be 3 digits. But we're only going to use the first 2.